build.gradle

```diff
@@ -15,10 +15,14 @@ repositories {
 
 dependencies {
     implementation 'javax.servlet:javax.servlet-api:4.0.1'
-    implementation 'org.eclipse.jetty:jetty-server:11.0.8'
     testImplementation 'junit:junit:4.+'
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.2'
     testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.8.2'
+    implementation 'org.json:json:20211205'
+    implementation 'org.springframework:spring-test:5.3.15'
+    implementation 'org.eclipse.jetty:jetty-server:9.4.43.v20210629'
```

Comment:
any specific reason why we pick an older version of jetty?

Reply:
Originally after I removed the requirement of java 11 the previous newer version of jetty won't compile using gradle and throws error: [Unable to compile: class file has wrong version 55.0, should be 52.0], and after I lowered the version of it, it builds